packages/cli/test/integration/me.api.test.ts

```diff
@@ -161,37 +157,56 @@ describe('Owner shell', () => {
 		}
 	});
 
-	test('POST /me/api-key should create an api key', async () => {
-		const response = await authOwnerShellAgent.post('/me/api-key');
+	test('POST /me/api-keys should create an api key', async () => {
```

Comment:
Since the api key endpoints have their own prefix, maybe we could move them to a separate file? There's already quite a bit of stuff in this file

Reply:
Next PR will be extract away the api-keys path into it's own controller. I do not think we need to keep it under the `/me` path.